cmd/genesis.go

```diff
@@ -96,7 +80,135 @@ var verifyCmd = &cobra.Command{
 	},
 }
 
+var processCSVCmd = &cobra.Command{
+	Use: "csv",
+	Short: "Process a CSV file with account definitions " +
+		"(name, amount, key1, key2, key3, key4, key5, required_signatures)",
```

Comment:
IIRC we actually support up to ten keys, although in practice I don't think anyone is using that many. Maybe worth adding to the description something like "(for an account with up to five pubkeys)"?